decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative('./name_able')
+
+class Decorator < Nameable
+  def initialize(name_able)
```

Comment:
- [ ] OPTIONAL
It's a best practice to mark `nameable` with `attr_accessor`, to give an indication that we intend to both _read_ and write this variable. Let's endeavor to improve upon the same.

Reply:
Thanks for the review
I implemented the optional suggestion